js/signin.js

```diff
@@ -0,0 +1,87 @@
+const idInput = document.querySelector('#id-label');
+const pwInput = document.querySelector('#password-label');
+const idErrorMsg = document.createElement('span');
+const pwErrorMsg = document.createElement('span');
+const inputBox = document.querySelector('.input-box:nth-child(2)');
+const idInvalidMsg = document.createElement('span');
+const loginBtn = document.querySelector('.login-btn');
+const eyeBtn = document.querySelector('.eye-off');
+
+let reg = /^[0-9a-zA-Z]([-_\.]?[0-9a-zA-Z])*@[0-9a-zA-Z]([-_\.]?[0-9a-zA-Z])*\.[a-zA-Z]{2,3}$/;
```

Comment:
`let` 보다는 `const`가 좋을 것 같아요!
그리고 상수는 Upper Case로 선언해주는 것이 가독성이 좋답니다.

저라면 아래와 같이 할 것 같네요!

```suggestion
const REG_EMAIL = /^[0-9a-zA-Z]([-_\.]?[0-9a-zA-Z])*@[0-9a-zA-Z]([-_\.]?[0-9a-zA-Z])*\.[a-zA-Z]{2,3}$/;
```